kt/godot-library/src/main/kotlin/godot/core/StringName.kt

```diff
@@ -36,7 +36,7 @@ class StringName : NativeCoreType {
         return TransferContext.readReturnValue(VariantType.STRING, false) as String
     }
 
-    //TODO/4.0: Implement
+    operator fun <T> invoke(stringOperation: String.() -> T): T = toString().stringOperation()
```

Comment:
Not sure what I'm looking at here. What would a typical use case of this invoke operator look like?

Reply:
StringName has a LOOOOOOT of functions that would be cumbersome to reimplement, even as Bridge, and they happen to be the same as the regular Godot String. So instead of spending time implementing that, we decide to just add a utility function that can transform a StringName using the regular Kotlin String methods.

Reply:
hmm i see. but then i would a dd a documentation on the function with a link to [this page](https://docs.godotengine.org/en/latest/classes/class_stringname.html) and a hint that one can reimplement those functions with this operator, and probably also provide an example of such a reimplementation using this operator